[2025-02-11T00:00:20.364+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T00:00:20.394+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: web_to_snowflake.create_file_format scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T00:00:20.409+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: web_to_snowflake.create_file_format scheduled__2025-02-10T00:00:00+00:00 [queued]>
[2025-02-11T00:00:20.412+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-11T00:00:20.440+0000] {taskinstance.py:2889} INFO - Executing <Task(SnowflakeOperator): create_file_format> on 2025-02-10 00:00:00+00:00
[2025-02-11T00:00:20.455+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=219) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-11T00:00:20.461+0000] {standard_task_runner.py:72} INFO - Started process 221 to run task
[2025-02-11T00:00:20.458+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'web_to_snowflake', 'create_file_format', 'scheduled__2025-02-10T00:00:00+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/web_to_snowflake.py', '--cfg-path', '/tmp/tmpme4h_x25']
[2025-02-11T00:00:20.465+0000] {standard_task_runner.py:105} INFO - Job 73: Subtask create_file_format
[2025-02-11T00:00:20.575+0000] {task_command.py:467} INFO - Running <TaskInstance: web_to_snowflake.create_file_format scheduled__2025-02-10T00:00:00+00:00 [running]> on host 48d23ee18b74
[2025-02-11T00:00:20.717+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='web_to_snowflake' AIRFLOW_CTX_TASK_ID='create_file_format' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-10T00:00:00+00:00'
[2025-02-11T00:00:20.720+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-11T00:00:20.743+0000] {sql.py:278} INFO - Executing: 
    CREATE OR REPLACE FILE FORMAT my_csv_format
        TYPE = 'CSV'
        FIELD_DELIMITER = ','
        SKIP_HEADER = 1
        NULL_IF = ('NULL', 'null')
        EMPTY_FIELD_AS_NULL = TRUE
        DATE_FORMAT = 'YYYY-MM-DD';
    
[2025-02-11T00:00:20.766+0000] {base.py:84} INFO - Retrieving connection 'snowflake_default'
[2025-02-11T00:00:20.790+0000] {base.py:84} INFO - Retrieving connection 'snowflake_default'
[2025-02-11T00:00:20.793+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-02-11T00:00:20.796+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2025-02-11T00:00:20.798+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-02-11T00:00:21.673+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-02-11T00:00:21.676+0000] {sql.py:553} INFO - Running statement: CREATE OR REPLACE FILE FORMAT my_csv_format
        TYPE = 'CSV'
        FIELD_DELIMITER = ','
        SKIP_HEADER = 1
        NULL_IF = ('NULL', 'null')
        EMPTY_FIELD_AS_NULL = TRUE
        DATE_FORMAT = 'YYYY-MM-DD';, parameters: None
[2025-02-11T00:00:21.912+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-02-11T00:00:21.914+0000] {sql.py:562} INFO - Rows affected: 1
[2025-02-11T00:00:21.916+0000] {snowflake.py:447} INFO - Rows affected: 1
[2025-02-11T00:00:21.918+0000] {snowflake.py:448} INFO - Snowflake query id: 01ba4f40-3202-c1d0-0005-f7120002509e
[2025-02-11T00:00:21.993+0000] {cursor.py:1166} INFO - Number of results in first chunk: 1
[2025-02-11T00:00:21.995+0000] {connection.py:789} INFO - closed
[2025-02-11T00:00:22.047+0000] {connection.py:795} INFO - No async queries seem to be running, deleting session
[2025-02-11T00:00:22.176+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-11T00:00:22.179+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=web_to_snowflake, task_id=create_file_format, run_id=scheduled__2025-02-10T00:00:00+00:00, execution_date=20250210T000000, start_date=20250211T000020, end_date=20250211T000022
[2025-02-11T00:00:22.252+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T00:00:22.304+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-11T00:00:22.310+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
